openedx/features/badging/tests/test_model_badge.py

```diff
@@ -39,77 +29,10 @@ def test_save_badge_normal(self):
                       date_created=None)
         self.assertEqual(badge.save(), None)
 
-    def test_unearned_badges_empty(self):
+    def test_get_badges_json(self):
         """
-        Get unearned badges dictionary without saving any
-        badges earlier so nothing matches
+        Test get badges json method
```

Comment:
Lets update doc string stating the scenario and how data is generated

Reply:
done